keyhive_wasm/src/js/capability.rs

```diff
@@ -34,3 +35,23 @@ impl Capability {
         self.proof.dupe().into()
     }
 }
+
+#[wasm_bindgen]
+#[derive(Debug, Clone, Dupe)]
+pub struct SimpleCapability {
```

Comment:
Why did you need this over `Capability` which has the same plus one method? Were you running into memory issues?